.github/workflows/cancel-previous-workflow.yml

```diff
@@ -3,11 +3,15 @@ on:
   workflow_run:
     workflows: ["Tests", "PR Build", "Colima tests", "Container tests", "Check docs", "golangci-lint", "Master branch build/release (signed)", "CodeQL"]
     types:
-    - requested
+      - requested
+
 jobs:
   cancel:
+    name: Cancel Previous Runs
+    permissions:
+      actions: write
     runs-on: ubuntu-latest
     steps:
-    - uses: styfle/cancel-workflow-action@0.11.0
-      with:
-        workflow_id: all
+      - uses: styfle/cancel-workflow-action@0.11.0
```

Comment:
Different yaml editors seem to disagree about the placement of array items, some put at top level, some indented. In general, it's better not to change things that don't relate to the PR being worked on. I'm OK with this, but if we're going to insist on indentation, we should agree on a standard.

Reply:
I had other changes in this file before but this change should also be reverted, true.